src/style/pages/_schedule.scss

```diff
@@ -6,31 +6,61 @@
             margin-left: 15px;
 
             // for Day 1, Day 2 etc.
+            .planet {
+              transform: scaleY(-1); // Flips into correct orientation
+              float: left;
+              margin-right: 10px;
+              margin-top: 8px;
+            }
+
             h1 {
                 padding-left: 0px;
-                font-size: 3.8rem;
+                font-size: 3rem;
+                white-space: nowrap;
             }
 
             h4 {
                 color: $accent-2;
+                translate: 152px;
             }
 
             h4, span {
-                font-size: 2.5rem;
+                font-size: 1.7rem;
                 font-family: $header-font;
             }
 
+            .location {
+              display: flex;
+              translate: 152px;
+              margin-bottom: 30px;
+            }
+
             .day-info {
+
                 h2, h4 {
                     font-family: $body-font;
-                    font-size: 2.5rem;
+                    font-size: 1.5rem;
+                    translate: 152px;
+                    margin: auto;
+                    margin-bottom: -20px;
+                }
+                
+                h2 {
```

Comment:
These h2 and h4 styling changes are the ones that are affecting the home page, but that's because the home page also uses `.schedule` and `.day-info`. We can leave this as it is for now and just try to make class names specific or remove any unnecessary classes, so if day-info is no longer needed as a tag or class, we don't have to wrap these inside `.day-info`. Change as you see fit!